pkgs/by-name/pa/paperjam/package.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchgit
+, qpdf
+, libiconv
+, libpaper
+, asciidoc
+,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "paperjam";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "git://git.ucw.cz/paperjam.git";
+    # this is v1.2 + build fix
```

Comment:
then you should use the unstable format for version

Reply:
I used "1.2-unstable-2023-05-13", hope that is what you expected.